modules/agent-framework/connection-service/src/main/resources/application.yml

```diff
@@ -11,4 +11,9 @@ airavata:
     url: scigap02.sciencegateways.iu.edu
     port: 9930
     truststore:
-      path: CHANGE_ME
+      path: /Users/lahiruj/Projects/airavata/lahiruj/modules/agent-framework/connection-service/src/main/resources/trustore.jks
```

Comment:
@lahirujayathilake add a gitignore for application.yml and add another file appication.yml.template.